systemtest/070-list-tags.bats

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bats
+#
+# list-tags tests
+#
+
+load helpers
+
+function setup() {
+    standard_setup
+
+    start_registry --enable-delete=true reg
+}
+
+# list from registry
+@test "list-tags: remove image from registry" {
+    local remote_image=docker://quay.io/libpod/busybox:latest
+    local img_tag=unsigned
+    local localimg=docker://localhost:5000/busybox:${img_tag}
+    local output=
+
+    run_skopeo copy --dest-tls-verify=false $remote_image $localimg
+    output=$(run_skopeo inspect --tls-verify=false --raw $localimg)
```

Comment:
Please don't do this. `run_skopeo` already sets `$output`, you do not need to do the `output=` thing.